tests/neighbor_classifier.py

```diff
@@ -0,0 +1,13 @@
+import unittest
+from src import neighbor_read
+from src import NeighborsClassifier
+import numpy as np
+
+
+class NeighborsClassifierTest(unittest.TestCase):
+
+    def test_cost_function(self):
+        input_train, input_test, target_train, target_test = neighbor_read.load_data()
+        model = NeighborsClassifier().model
+        model.fit(input_train, target_train)
+        self.assertEqual(target_train[0], 1)
```

Comment:
You check if the first value in the target column is one. 
Not a real test, you should check for model performance, and output closeness:
https://numpy.org/doc/stable/reference/generated/numpy.isclose.html